clash-lib/src/Clash/Driver.hs

```diff
@@ -325,9 +328,10 @@ generateHDL reprs bindingsMap hdlState primMap tcm tupTcm typeTrans eval
       -- See [Note] Create HDL dir before netlist generation
       let dir = hdlDir </> maybe "" t_name annM </> Data.Text.unpack modName'
       prepareDir (opt_cleanhdl opts) (extension hdlState2) dir
+      let topEnts1 = map ((\(Just t) -> t) . flip lookupVarEnv transformedTopEntities . topId) topEnts0
```

Comment:
Don't you mean, `fmap fromJust` :eyes:

Reply:
I didn't want to create a situation where we'd see `*** Exception: Maybe.fromJust: Nothing` if this logic ever breaks. At least an incomplete pattern match gives you a line number. (Although HasCallStack might save the day nowadays?)

Reply:
I would hope it would, but who knows :man_shrugging: 
I tend to prefer using `fromJust` directly because it's (1) a little less noisy and (2) makes `hlint` happier. Pros and cons.

Reply:
use `maybe (error “internal error”)`, you’ll get the callstack then. Or use `$(curLoc)` just to be sure.